IoTuring/Entity/Deployments/Fanspeed/Fanspeed.py

```diff
@@ -0,0 +1,229 @@
+import psutil
+
+from IoTuring.Entity.Entity import Entity
+from IoTuring.Entity.EntityData import EntitySensor
+from IoTuring.Entity.ValueFormat import ValueFormatterOptions
+from IoTuring.MyApp.SystemConsts import OperatingSystemDetection as OsD
+from IoTuring.Configurator.MenuPreset import MenuPreset
+
+
+supports_win_fanspeed = False
+supports_linux_fanspeed = True
+supports_macos_fanspeed = False
+
+KEY_FANSPEED = "fanspeed"
+KEY_FANLABEL = "fanlabel"
+
+FALLBACK_PACKAGE_LABEL = "controller"
+FALLBACK_SENSOR_LABEL = "fan"
+
+CONFIG_KEY_CONTROLLER = "controller"
+
+FANSPEED_DECIMALS = 0
+
+
+class Fanspeed(Entity):
+    """Entity to read fanspeed"""
+
+    NAME = "Fanspeed"
+
+    def Initialize(self) -> None:
+        """Initialize the Class, setup Formatter, determin specificInitialize and specificUpdate depending on OS"""
+        self.fanspeedFormatOptions = ValueFormatterOptions(
+            value_type=ValueFormatterOptions.TYPE_ROTATION, decimals=FANSPEED_DECIMALS
+        )
+
+        self.specificInitialize = None
+        self.specificUpdate = None
+
+        if OsD.IsLinux():
+            self.specificInitialize = self.InitLinux
+            self.specificUpdate = self.UpdateLinux
+        self.specificInitialize()
+
+    def InitLinux(self) -> None:
+        """OS dependant Init for Linux"""
+        self.configuredPackages: list[str] = []
+        self.registeredPackages: list[psutilFanspeedPackage] = []
+        sensors = psutil.sensors_fans()
+        self.Log(self.LOG_DEBUG, f"fancontrollers found:{sensors}")
+        # load all controllers from config
+        self.configuredPackages.append(
+            self.GetConfigurations()[CONFIG_KEY_CONTROLLER]
+        )  
+        self.Log(self.LOG_DEBUG, f"registered controllers from config:{self.registeredPackages}")
+        
+        for controllerName,data, in sensors.items():  # read the controllernames and fanspeeds
+            # build packages from controllernames if they are in the config
+            if controllerName in self.configuredPackages:
+                package = psutilFanspeedPackage(controllerName, data)
+                # register the entity and give it an initial value
+                if package.hasCurrent():
+                    # build the list of FanspeedPackage objects and register them as Entitys
+                    self.registeredPackages.append(package)
+                    self.RegisterEntitySensor(
+                        EntitySensor(
+                            self,
+                            package.packageName,
+                            supportsExtraAttributes=True,
+                            valueFormatterOptions=self.fanspeedFormatOptions,
```

Comment:
In order to change the entity value unit, the unit is set while registering the entity

Reply:
So if you don't want anymore the rpm there, just remove the value formatter option here